spatial-lib/test/cmr/spatial/test/arc.clj

```diff
@@ -1,20 +1,21 @@
 (ns cmr.spatial.test.arc
-  (:require [clojure.test :refer :all]
-            ; [clojure.test.check.clojure-test :refer [defspec]]
-            ;; Temporarily included to use the fixed defspec. Remove once issue is fixed.
-            [cmr.common.test.test-check-ext :refer [defspec]]
-
-            [clojure.test.check.properties :refer [for-all]]
-            [clojure.test.check.generators :as gen]
-
-            ;; my code
-            [cmr.spatial.math :refer :all]
-            [cmr.spatial.arc :as a]
-            [cmr.spatial.point :as p]
-            [cmr.spatial.vector :as v]
-            [cmr.spatial.mbr :as mbr]
-            [cmr.spatial.test.generators :as sgen]))
-
+  (:require
+   [clojure.test :refer [are deftest is testing]]
+   ; [clojure.test.check.clojure-test :refer [defspec]]
+   ;; Temporarily included to use the fixed defspec. Remove once issue is fixed.
```

Comment:
reminder to remove

Reply:
that is actually not mine, I don't know what this is about, but many of the files have it. I'm not willing to do bigger changes to this library without understanding the math that is inside it.